ur5_info/JointPositions[] positions
float64[] delay
float64 gripperAngle